docs/data/osdf/install-origin-rpm.md

```diff
@@ -0,0 +1,231 @@
+    * An origin in the OSDF should have at least:
+        * 1 core
+        * 1 Gbps connectivity
+        * 12 GB of RAM
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Obtain root access to the host
+* Prepare [the required Yum repositories](../../common/yum.md);
+  the [OSG 24 repositories](../../common/yum.md#install-the-osg-repositories) should be used.
+
+!!! note "Host certificates"
+    Origins are accessed by users through browsers, meaning origins need a certificate from a CA acceptable to a standard browser.
+    Examples include [Let's Encrypt](../../security/host-certs/lets-encrypt.md) or the InCommon RSA CA.
+    Origins without a valid certificate for the browser cannot be added to the OSDF.
+    Note that, unlike legacy grid software, the public certificate file will need to contain the "full chain", including any
+    intermediate CAs (if you're unsure about your setup, try accessing your origin from your browser).
+    
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
+
+Installing the Origin
+---------------------
+
+The origin service is provided by the `osdf-origin` RPM.
+Install it using the following command:
+
+
+OSG 24:
+```console
+root@host # yum install osdf-origin
+```
+
+
+!!! note "osdf-origin 7.11.1"
+    This document covers versions 7.11.1 and later of the `osdf-origin` package; ensure the above installation
+    results in an appropriate version.
+
+Configuring the Origin Server
+-----------------------------
+
+Create a file named `/etc/pelican/config.d/20-origin.yaml`
+
+You must tell Pelican the data to export to the federation.
+An origin may export one or more directory trees, or one or more S3 buckets -- follow one of the sections below.
+A single origin cannot export both a bucket and a directory tree.
+
+
+
+### Configuring POSIX (directory) export
+
+Set these options to export one or more directory trees to the federation.
+
+```
+Origin:
+  StorageType: "posix"
+  Exports:
+    # You may have one or more of the following block:
+    - FederationPrefix: "<EXTERNAL OSDF NAMESPACE>"
+      StoragePrefix:    "<LOCAL FILESYSTEM DIRECTORY>"
+      Capabilities:    # Add or remove as desired
+        - Reads        # Enable authenticated reading of objects from under the directory tree through a cache
+        - PublicReads  # Enable unauthenticated reading of objects from under the directory tree through a cache
+        - DirectReads  # Enable reading objects from under the directory tree without going through a cache
+        - Listings     # Enable directory listings
+        - Writes       # Enable writing to files in the directory tree
```

Comment:
As we discussed, we should turn this into example config in the packaging. I think I remembered putting this together https://github.com/opensciencegrid/tiger-osg-config/blob/master/manifests/base/pelican-origin-yaml-config/50-instance.yaml

Reply:
Actually, we really should just use what's in upstream. I forget why we didn't just do that

- https://github.com/PelicanPlatform/pelican/blob/main/systemd/examples/20-origin-exports.yaml
- https://github.com/PelicanPlatform/pelican/blob/main/systemd/examples/60-origin-multiuser.yaml

Reply:
As we discussed, I'll take care of that in a future PR.